docs/design/fabric-v2+/interfaces.md

```diff
@@ -27,7 +27,7 @@ func Invoke(stub shim.ChaincodeStubInterface) pb.Response {}
 // Note: to get the endpoints of FPC endorsing peers do the following:
 // - discover all endorsing peers (and their endpoints) for the FPC chaincode using "normal" lifecycle
 // - query `getEnclaveId` at all the peers discovered
-// - query queryCCInfo with all received enclave_ids
+// - query `queryListEnclaveCredentials` with all received enclave_ids
```

Comment:
Maybe we could drop here all protobuf definitions and just reference the .proto files?  Removes any duplication?